pkg/apis/integreatly/v1alpha1/grafanadashboard_types.go

```diff
@@ -74,7 +74,7 @@ func init() {
 	SchemeBuilder.Register(&GrafanaDashboard{}, &GrafanaDashboardList{})
 }
 
-//nolint
+// nolint
```

Comment:
Can you revert this change. I think it's caused by a golang version mismatch with the already committed code. For 8.5 we probably want to upgrade to a more recent go version etc, so let's do that all in one place.

Reply:
yep, something in the make file is changing this and the next file... it is so annoying